Geometry/RPCGeometryBuilder/src/RPCGeometryParsFromDD.cc

```diff
@@ -1,31 +1,38 @@
-/** Implementation of the RPC Geometry Builder from DDD
+/* Implementation of the  RPCGeometryParsFromDD Class
+ *  Build the RPCGeometry from the DDD and DD4Hep description
+ *  
+ *  DD4hep part added to the original old file (DD version) made by M. Maggi (INFN Bari)
+ *  Author:  Sergio Lo Meo (sergio.lo.meo@cern.ch) 
+ *  Created:  Mon, 09 Nov 2020 
  *
- *  \author Port of: MuDDDRPCBuilder (ORCA)
- *  \author M. Maggi - INFN Bari
  */
 #include "Geometry/RPCGeometryBuilder/src/RPCGeometryParsFromDD.h"
 #include "DataFormats/MuonDetId/interface/RPCDetId.h"
-
 #include <CondFormats/GeometryObjects/interface/RecoIdealGeometry.h>
 #include <DetectorDescription/Core/interface/DDFilter.h>
 #include <DetectorDescription/Core/interface/DDFilteredView.h>
 #include <DetectorDescription/Core/interface/DDSolid.h>
-
 #include "Geometry/MuonNumbering/interface/MuonGeometryNumbering.h"
 #include "Geometry/MuonNumbering/interface/MuonBaseNumber.h"
 #include "Geometry/MuonNumbering/interface/RPCNumberingScheme.h"
-
 #include "DataFormats/GeometryVector/interface/Basic3DVector.h"
-
 #include "CLHEP/Units/GlobalSystemOfUnits.h"
-
 #include <iostream>
 #include <algorithm>
+#include <DetectorDescription/DDCMS/interface/DDFilteredView.h>
+#include <DetectorDescription/DDCMS/interface/DDCompactView.h>
+#include "DetectorDescription/DDCMS/interface/DDSpecParRegistry.h"
+#include "DataFormats/Math/interface/CMSUnits.h"
```

Comment:
Why is `CMSUnits.h` included? I don't see anything from it used in this file.

Reply:
@cvuosalo : I deleted this #include